examples/jsonnet/bazel_init_test.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package jsonnet_test
+
+import (
+	"github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/testutil"
+)
+
+func init() {
+	testutil.ChangeToBazelDir("examples/jsonnet")
```

Comment:
This is the default testdir already..?

Reply:
no, unfortunately. The default testdir is the workspace root.

Reply:
> These files are available using the relative path path/to/data/file. In tests, it is also possible to refer to them by joining the paths of the test's source directory and the workspace-relative path, e.g. ${TEST_SRCDIR}/workspace/path/to/data/file.

From https://docs.bazel.build/versions/master/build-ref.html#data

"relative" indicates that the working dir is configured properly. Do the tests pass if you remove this file?

Reply:
No. The tricky thing is I want the tests to pass both with Bazel and with normal `go test`. So, the solution here is what's used in `third_party` (`third_party_go_test`) have an explicit change-dir that's executed only for bazel paths so `ioutil` works for both `go test` and  `bazel test`